From: Wei Yongjun <weiyj.lk@gmail.com>
Date: Sat, 20 Aug 2016 15:31:05 +0000
Subject: clk: mvebu: Remove redundant dev_err call in
 armada_3700_periph_clock_probe()
Git-commit: 0f7dd7acda0add9cbd39caa6902c9e79ecb0a89d
Patch-mainline: v4.9-rc1
References: fate#319899 fate#320700

There is a error message within devm_ioremap_resource
already, so remove the dev_err call to avoid redundant
error message.

Signed-off-by: Wei Yongjun <weiyj.lk@gmail.com>
Signed-off-by: Stephen Boyd <sboyd@codeaurora.org>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/clk/mvebu/armada-37xx-periph.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/clk/mvebu/armada-37xx-periph.c b/drivers/clk/mvebu/armada-37xx-periph.c
index 5bb13c9..45905fc 100644
--- a/drivers/clk/mvebu/armada-37xx-periph.c
+++ b/drivers/clk/mvebu/armada-37xx-periph.c
@@ -383,10 +383,8 @@ static int armada_3700_periph_clock_probe(struct platform_device *pdev)
 
 	res = platform_get_resource(pdev, IORESOURCE_MEM, 0);
 	reg = devm_ioremap_resource(dev, res);
-	if (IS_ERR(reg)) {
-		dev_err(dev, "Could not map the periph clock registers\n");
+	if (IS_ERR(reg))
 		return PTR_ERR(reg);
-	}
 
 	driver_data = devm_kzalloc(dev, sizeof(*driver_data), GFP_KERNEL);
 	if (!driver_data)
-- 
2.10.0

